src/components/Pagination/Paginator.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Form, FormControl, FormGroup, ControlLabel } from '../Form';
+import { DropdownButton } from '../Button';
+import { ListGroup, ListGroupItem } from '../ListGroup';
+import { MenuItem } from '../MenuItem';
+import { Icon } from '../Icon';
+
+/**
+ * Paginator component for Patternfly React
+ */
+const Paginator = ({
+  className,
+  pagination,
+  amountOfPages,
+  itemCount,
+  itemsStart,
+  itemsEnd,
+  onPerPageSelect,
+  onFirstPage,
+  onPreviousPage,
+  onPageInput,
+  onNextPage,
+  onLastPage
+}) => {
+  const { page, perPage, perPageOptions } = pagination;
+
+  return (
+    <Form className={className}>
+      <FormGroup>
+        <DropdownButton
+          title={perPage}
+          dropup
+          onSelect={onPerPageSelect}
+          id="paginator-dropdown"
+        >
+          {perPageOptions &&
+            perPageOptions.length &&
+            perPageOptions.map((option, i) => {
+              return (
+                <MenuItem eventKey={option} active={option === perPage} key={i}>
+                  {option}
+                </MenuItem>
+              );
+            })}
+        </DropdownButton>
+        <span> per page </span>
+      </FormGroup>
+      <FormGroup>
+        <span>
+          <span className="pagination-pf-items-current">
+            {itemsStart}-{itemsEnd}
+          </span>
+          &nbsp;of&nbsp;
+          <span className="pagination-pf-items-total">{itemCount}</span>
+        </span>
+        <ListGroup componentClass="ul" bsClass="pagination pagination-pf-back">
+          <ListGroupItem
+            bsClass=" "
+            listItem
+            className={page === 1 ? 'disabled' : ''}
+          >
+            <a
+              href="#"
+              title="First Page"
```

Comment:
Any thought to use customizable messages?

Reply:
this is code should be gone now... renamed `Paginator.js` to `PaginationRow.js` and made sure all the strings were localizable. Let me know if I missing something though?